CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 * Require Xcode 15.0+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
+* Update Cardinal SDK to version 2.2.5-9
```

Comment:
Lets add this under a `BraintreeThreeDSecure` section with a note that it includes code signing and privacy manifest requirements

Reply:
for sure, addressed in 3439b8d94359bcc9bed284d094168a8bde1b6084